xml/System.Web.UI.DataVisualization.Charting/SeriesCollection.xml

```diff
@@ -56,7 +56,7 @@
   
  The `name` argument refers to the `Name` property of the object being added.  
   
- **Note** If you use this method to add <xref:System.Web.UI.DataVisualization.Charting.Series> objects to the <xref:System.Web.UI.DataVisualization.Charting.SeriesCollection> class, the associated <xref:System.Web.UI.DataVisualization.Charting.DataPoint> objects of the newly added <xref:System.Web.UI.DataVisualization.Charting.Series> object cannot have more than one Y-value per data point. To add a series that can handle multiple Y-values per data point, use the <xref:System.Web.UI.DataVisualization.Charting.SeriesCollection.Add%2A> System.Int32)?qualifyHint=False&autoUpgrade=True definition of this overloaded method.  
+ **Note** If you use this method to add <xref:System.Web.UI.DataVisualization.Charting.Series> objects to the <xref:System.Web.UI.DataVisualization.Charting.SeriesCollection> class, the associated <xref:System.Web.UI.DataVisualization.Charting.DataPoint> objects of the newly added <xref:System.Web.UI.DataVisualization.Charting.Series> object cannot have more than one Y-value per data point. To add a series that can handle multiple Y-values per data point, use the <xref:System.Web.UI.DataVisualization.Charting.SeriesCollection.Add(System.Int32)> definition of this overloaded method.  
```

Comment:
The link to Add(Int32) is broken; in fact, there is no such method. The only other add overload is inherited, so a link won't resolve unless you link to the base class members, System.Collections.ObjectModel.Collection<T>.Add(T).

Reply:
@rpetrusha, that's really strange, as I thought I looked up both of these.  I think it might be confusing to refer to System.Collections.ObjectModel.Collection.Add(T) in this regard without a larger discussion. I think it should just be removed. Agree?

Reply:
Yes, it's very confusing, @JRAlexander, especially when the base class has no immediately clear relationship to the derived class, as is the case here. I think that the best thing to do is to replace the link with a \`...\`.